app/domain/authentication/authn_jwt/input_validation/validate_claim_name.rb

```diff
@@ -34,7 +34,7 @@ def validate_claim_name_value
         end
         
         def valid_claim_name_regex
-          @valid_claim_name_regex ||= Regexp.new(VALID_CLAIM_NAME_REGEX)
```

Comment:
Please check if `VALID_CLAIM_NAME_REGEX` const is still in use.
If no please remove